tests/other/metadce/hello_world_O3_MAIN_MODULE.sent

```diff
@@ -643,7 +643,6 @@ emscripten_autodebug_i16
 emscripten_autodebug_i32
 emscripten_autodebug_i64
 emscripten_autodebug_i8
-emscripten_builtin_memalign
```

Comment:
Do you need to do this for the O1 test too? 

I'm seeing test_binaryen_metadce_hello_main_module_1 fail on wasm waterfall.